### ISSUE TITLE
pivot_wider with integer64

### ISSUE THREAD

**Issue by JohnCoene** [tidyverse/tidyr#758]
The data.frame returned by i.e.: the RPostgres package by @krlmlr DBI can return `*64` class objects [bit64](https://github.com/r-dbi/RPostgres/blob/master/DESCRIPTION): These seem to mess up the values with `pivot_wider`. 

```r
fish_encounters %>% 
  dplyr::mutate(
    seen = bit64::as.integer64(seen)
  ) %>% 
  tidyr::pivot_wider(
    names_from = station,
    values_from = seen
  )
```

Apologies if this should be posted under bit64 or RPostgres.

**Comment by jtbates** [comment]
Duplicate of #363 

The bug has been fixed in [bit64 version 0.9-8](https://r-forge.r-project.org/scm/viewvc.php/*checkout*/pkg/bit64/NEWS?revision=177&root=ff), but [CRAN is still on 0.9-7](https://cran.r-project.org/web/packages/bit64/index.html).

@joehl, could we get a CRAN release for 0.9-8?

**Comment by hadley** [comment]
Closing this issue since it seems like it needs to be resolved by a bit64 release.
